bucket/sd-card-formatter.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "5.0.2",
+    "description": "The SD Memory Card Formatter formats SD Memory Card, SDHC Memory Card and SDXC Memory Card (respectively SD/SDHC/SDXC Cards) complying with the SD File System Specification created by the SD Association (SDA).",
+    "homepage": "https://www.sdcard.org/downloads/formatter/",
+    "license": {
+        "identifier": "SDA",
```

Comment:
SDA is not a valid [SPDX](https://spdx.org/licenses/) identifier. Please change it to `Freeware`